### ISSUE TITLE
An Updates

### ISSUE THREAD

**Issue by jlkalberer** [falcordotnet/falcor.net#1]
Are you actually working on this or just keeping the actual source private?

**Comment by craigsmitham** [comment]
Hey John - actively working on this. Haven't pushed updates in awhile due to lots of churn. I'm targeting 10/26/15 for an early alpha release. Interested in contributing? 
Date: Sun, 11 Oct 2015 07:44:00 -0700

**Comment by jlkalberer** [comment]
Yeah, I would love to but I have other commitments at the moment.  I'll
definitely contribute when I can.

**Comment by craigsmitham** [comment]
Pushed latest bits here: https://github.com/falcordotnet/falcor.net

I’m working on building out the example that will have all the examples from https://github.com/Netflix/falcor-router-demo/blob/master/index.js

You can see my progress here: https://github.com/falcordotnet/falcor.net/blob/master/examples/Falcor.Examples.Netflix/NetflixRouter.cs

**Comment by jlkalberer** [comment]
Wooo. Nice.  I would love to use Falcor instead of REST.  I'll test this
out when I get some time.

**Comment by dotnetdude** [comment]
Hi  @craigsmitham Thanks for implementing Falcor in .net, Any progress on this one? Any plan to release a stable version?

**Comment by dotnetdude** [comment]
Okay. Ignore my Question. Just learned falcor is still in developer preview [https://www.npmjs.com/package/falcor](https://www.npmjs.com/package/falcor)

**Comment by craigsmitham** [comment]
@dotnetdude thanks for checking this project out. Yes, as Falcor.js is still in developer preview, I haven't made many continued investments in Falcor.NET. I'm still really happy of the work that is done here so far, and there is a really solid foundation here for any future implementation. Personally, I've been implementing a small-ish site using Angular2, .NET Core MVC, and ngrx/store (redux). It's a bit of a different approach from Falcor. I'm still learning. I think some mix of GraphQL, Falcor, and Redux is going to be the holy grail for managing client/server state in the future. Until then, each have their own unique trade-offs. Having worked with the redux pattern (ngrx/store) for about 6 months now, I see its key benefits but also see where Falcor provides some key capabilities as well. No time to dig into observations now, but if you are interested in Falcor - I would encourage you to keep diving in, including Falcor.NET.
